playwright/src/test/java/com/microsoft/playwright/TestBrowserTypeConnect.java

```diff
@@ -34,16 +36,19 @@
 import java.util.concurrent.ExecutionException;
 import java.util.stream.Collectors;
 
+import static com.microsoft.playwright.TestOptionsFactories.isWindows;
 import static com.microsoft.playwright.Utils.*;
 import static java.nio.charset.StandardCharsets.UTF_8;
 import static java.util.Arrays.asList;
 import static org.junit.jupiter.api.Assertions.*;
 
-public class TestBrowserTypeConnect extends TestBase {
+@FixtureTest
+@UsePlaywright(TestBrowserTypeConnect.ConnectOptionsFactory.class)
+public class TestBrowserTypeConnect {
   static Path FILE_TO_UPLOAD = Paths.get("src/test/resources/file-to-upload.txt");
 
-  private Process browserServer;
-  private String wsEndpoint;
+  private static Process browserServer;
```

Comment:
What's the rationale for changing these to public? These tests are independent and (at least in theory) could run in parallel but if we the browserServer/endpoint are shared in a static field, it may lead to races (admittedly, not in the current mode where we don't parallelize within class).

Reply:
do you mean why did I make them static?  BeforeAll/AfterAll are supposed to be static by default and because these are set/closed in those methods, I made these static also.  https://junit.org/junit5/docs/current/user-guide/#writing-tests-test-instance-lifecycle

Reply:
I can add `@TestInstance(Lifecycle.PER_CLASS)` to this test class and change the fields back to being instance fields if you'd like.  I was trying to keep as much default junit functionality as possible.  As far as supporting per-method parallelization goes, we would either keep these as static or we would be launching/closing the browser in `BeforeEach/AfterEach` hooks.

Reply:
It was inheriting  `@TestInstance(Lifecycle.PER_CLASS)` from `TestBase` before, so I'd just add the annotation explicitly while switching to `@UsePlaywright`.